eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -9,6 +9,10 @@ steps:
       git checkout 564ad63ae72d18422533fa1da9d396e7703c1cb5
     displayName: Setup Identity Resolver
 
+  - pwsh: |
```

Comment:
Did you have this for debugging? I'm not sure what you are trying to do with this step.

Reply:
If you use `$(OwningGHUser)` further on, pshell recognizes that as an actual _action_ to be taken, and crashes.

Setting it to _nothing_ explicitly ELIMINATES that error further along, while remaining super straightforward as far as code readability. The reason I do it _here_ instead of in a `variables` declaration, is because this is a callable template, and doesn't make assumptions about the what variables have been set prior to running this thing.

Reply:
Nope. if it doesn't map, it'll resolve to nothing. This is all good. Won't be present after a push.